k8s/configmap.yaml

```diff
@@ -0,0 +1,124 @@
+apiVersion: v1
+kind: ConfigMap
```

Comment:
also, maybe there is a better name of this file? e.g. `validator-setup-configmap.yml` or something

Reply:
Well the ConfigMap can technically have multiple things in it since its just a key-value store. If we had multiple scripts they could live in a single ConfigMap if needed, hence a generic name.  Maybe since this is specific to helium validators and we wouldn't use it for anything else what about `helium-validators-configmap.yml`. I feel like `setup` in the name is misleading because the script runs the entire validator, its more than a setup script.

Reply:
sounds good to me